docs/SUMMARY.md

```diff
@@ -12,6 +12,10 @@
   - [Transaction Routing](./architecture/how/tx_routing.md)
 - [Trie](./architecture/trie.md)
 - [Network](./architecture/network.md)
+- [Gas Cost Parameters](./architecture/gas_params/README.md)
```

Comment:
Paths here are visible in urls, so I think it's marginally better to stick to short, catchy names. In this case, `gas` would be a bit better than `gas_parms`.